Lib/fontTools/feaLib/builder_test.py

```diff
@@ -361,6 +363,92 @@ def test_lookup_inside_feature_aalt(self):
             "Lookup blocks cannot be placed inside 'aalt' features",
             self.build, "feature aalt {lookup L {} L;} aalt;")
 
+    def test_extensions(self):
+        class ast_BaseClass(ast.MarkClass):
+            def asFea(self, indent=""):
+                return ""
+
+        class ast_BaseClassDefinition(ast.MarkClassDefinition):
+            def asFea(self, indent=""):
+                return ""
+
+        class ast_MarkBasePosStatement(ast.MarkBasePosStatement):
+            def asFea(self, indent=""):
+                if isinstance(self.base, ast.MarkClassName) :
```

Comment:
sorry to be such a pain, but would you mind deleting those spaces before the colon?
It's not consistent with the rest of the code.